server/events/runtime/init_step_runner.go

```diff
@@ -16,12 +16,12 @@ func (i *InitStepRunner) Run(ctx models.ProjectCommandContext, extraArgs []strin
 	if ctx.TerraformVersion != nil {
 		tfVersion = ctx.TerraformVersion
 	}
-	terraformInitCmd := append([]string{"init", "-input=false", "-no-color", "-upgrade"}, extraArgs...)
+	terraformInitCmd := append([]string{"init", "-input=false", "-no-color"}, extraArgs...)
 
 	// If we're running < 0.9 we have to use `terraform get` instead of `init`.
 	if MustConstraint("< 0.9.0").Check(tfVersion) {
 		ctx.Log.Info("running terraform version %s so will use `get` instead of `init`", tfVersion)
-		terraformInitCmd = append([]string{"get", "-no-color", "-upgrade"}, extraArgs...)
+		terraformInitCmd = append([]string{"get", "-no-color"}, extraArgs...)
```

Comment:
General philiosophy of Atlantis is to not make breaking changes unless absolutely necessary.  In this case, there are tons of people on older terraform versions. Not feasible to just force an upgrade/code change on everyone.

Version checks are already happening in this function, so it doesn't seem like a huge deal to just dynamically build the args based on the version.